bin/custom/questions/src/com/expertsoft/populators/ProductQuestionsPopulator.java

```diff
@@ -0,0 +1,34 @@
+package com.expertsoft.populators;
+
+import com.expertsoft.questions.model.QuestionModel;
+import de.hybris.platform.commercefacades.product.data.ProductData;
+import de.hybris.platform.converters.Populator;
+import de.hybris.platform.core.model.product.ProductModel;
+import de.hybris.platform.servicelayer.dto.converter.ConversionException;
+import de.hybris.platform.servicelayer.dto.converter.Converter;
+import questions.data.QuestionData;
+
+import javax.annotation.Resource;
+import java.util.stream.Collectors;
+
+public class ProductQuestionsPopulator implements Populator<ProductModel, ProductData> {
+    @Resource(name = "questionModel2DataConverter")
```

Comment:
Do not mix xml- and annotation-based spring configs, usually in hybris converters/populators are xml-configured beans